charts/opencost/templates/ingress.yaml

```
@@ -37,4 +37,45 @@ spec:
                   name: {{ $.Values.opencost.ui.ingress.servicePort }}
         {{- end }}
   {{- end }}
+{{- end }}
+---
+{{- if .Values.opencost.exporter.apiIngress.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "opencost.fullname" . }}-apiIngress
```

This name is not valid in kubernetes
```suggestion
  name: {{ include "opencost.fullname" . }}-api-ingress
```